lib/active_record/connection_adapters/cockroachdb/schema_statements.rb

```diff
@@ -68,14 +68,19 @@ def pk_and_sequence_for(table) # :nodoc:
                  pg_attribute  attr,
                  pg_depend     dep,
                  pg_constraint cons,
-                 pg_namespace  nsp
+                 pg_namespace  nsp,
+                 -- TODO: use the pg_catalog.pg_attribute(attishidden) column when it's added instead of joining on crdb_internal.
```

Comment:
it could be useful to link to https://github.com/cockroachdb/cockroach/pull/126397 in the TODO, so we can easily see when the new column is in a CRDB release.